controller/src/test/java/io/pravega/controller/store/stream/TableHelperTest.java

```diff
@@ -687,43 +598,6 @@ public void predecessorAndSuccessorTest() {
         assertEquals(successors, new ArrayList<>());
     }
 
-    @Test
```

Comment:
Are we not creating an equivalent test case for the new flow?

Reply:
This test essentially used to verify "isScaleOngoing" and "isScaleRerun" in different scenarios. 
However, with epoch transition node, we don have these methods in tableHelper anymore. 
So this test is removed. 

Now the check for isScale ongoing is done via epochtransition node's presence and its equivalence.